frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/top-consumers-card/utils.ts

```diff
@@ -36,9 +40,6 @@ type GetStats = (
   unit?: string,
 ) => DataPoint[][];
 
-type SortResourcesProps = (
-  a: PrometheusResponse['data']['result'],
-  b: PrometheusResponse['data']['result'],
-) => number;
+type SortResourcesProps = (a: PrometheusResult, b: PrometheusResult) => number;
```

Comment:
Why this ?

Reply:
It fixes some type errors

Reply:
The types were not proper for this function. PrometheusResponse['data']['result'] is a list of results and we were passing the single values in it and due to usage of lodash anything was getting accepted and so it was working and not throwing error.